npool/miningpool/gw/v1/fraction/fraction.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+
+package miningpool.gateway.fraction.v1;
+
+option go_package = "github.com/NpoolPlatform/message/npool/miningpool/gw/v1/fraction";
+
+import "google/api/annotations.proto";
+import "npool/basetypes/miningpool/v1/enums.proto";
+
+// Service Name
+service Gateway {
+  rpc CreateFraction(CreateFractionRequest) returns (CreateFractionResponse) {
+    option (google.api.http) = {
+      post : "/v1/create/fraction"
+      body : "*"
+    };
+  }
+  rpc GetFraction(GetFractionRequest) returns (GetFractionResponse) {
+    option (google.api.http) = {
+      post : "/v1/get/fraction"
+      body : "*"
+    };
+  }
+  rpc GetUserFractions(GetUserFractionsRequest)
+      returns (GetUserFractionsResponse) {
+    option (google.api.http) = {
+      post : "/v1/get/user/fractions"
+      body : "*"
+    };
+  }
+}
+
+message Fraction {
+  uint32 ID = 10;
+  string EntID = 20;
+  string AppID = 30;
+  string UserID = 40;
+  string OrderUserID = 50;
+  basetypes.miningpool.v1.WithdrawState WithdrawState = 60;
+  uint32 WithdrawTime = 70;
+  uint32 PayTime = 80;
+  string Msg = 90;
+  uint32 CreatedAt = 100;
+  uint32 UpdatedAt = 110;
+}
+
+message CreateFractionRequest {
+  string AppID = 10;
+  string UserID = 20;
+  string OrderUserID = 30;
```

Comment:
I think we should use OrderID here, or this API should be implemented by order-gateway

Reply:
should be implemented by order-gateway